.github/workflows/run-all-tests.yml

```diff
@@ -10,6 +10,22 @@ on:
 
 jobs:
 
+  test_junit_springboot:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up JDK
+        uses: actions/setup-java@v3
+        with:
+          distribution: zulu
+          java-version: 11
+      - name: Build and run tests
+        # cd to the project directory and run the shell script
```

Comment:
This comment seems a bit redundant. We could mention that the project is built and tested with Maven instead, which explains why this is a separate job.

Reply:
Done!